<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TLoginDlg</class>
 <widget class="QDialog" name="TLoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>256</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>36</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="../res/request.qrc">
     <normaloff>:/images/cancel</normaloff>:/images/cancel</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>12</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="icon">
    <iconset resource="../res/request.qrc">
     <normaloff>:/images/ok</normaloff>:/images/ok</iconset>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>8</y>
     <width>128</width>
     <height>88</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>User</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="userEdit">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>24</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>44</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>64</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <zorder>label_11</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>userEdit</zorder>
   <zorder>label</zorder>
   <zorder>passwordEdit</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>userEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>okBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources>
  <include location="../res/request.qrc"/>
 </resources>
 <connections/>
</ui>
